Makefile

```diff
@@ -58,13 +73,18 @@ docs-clean:
 
 ################################################################################
 
-$(TARGET):
+$(TARGET): $(RUST_TARGET)
 	# Build the target and fail for errors and warnings. Ignore warnings
 	# having "qsave(strip_failed(..." that occur on macOS.
 	$(SWIPL) -t 'main,halt.' -O -q -f src/bootstrap.pl 2>&1 | \
 	  grep -v 'qsave(strip_failed' | \
 	  (! grep -e ERROR -e Warning)
 
+$(RUST_TARGET): $(RUST_FILES)
+	echo $(RUST_FILES)
```

Comment:
Is this for debugging in the future?

Reply:
no, that's an accidental echo I left in :). I'll remove it.